src/components/shared/navbar/Navbar.tsx

```diff
@@ -1,43 +1,34 @@
-import autobind from "autobind-decorator";
 import classNames from "classnames";
 import * as React from "react";
-
 import IconButton from "../icon-button/IconButton";
-
 import "./Navbar.scss";
 
-interface INavbarProps {
+interface NavbarProps {
   backButton?: boolean;
   title?: string;
   onBackButtonClick?: () => void;
   className?: string;
 }
 
-@autobind
-class Navbar extends React.Component<INavbarProps, unknown> {
-  public render() {
-    const { backButton, title, className } = this.props;
-
-    return (
-      <div className={classNames("navbar", className)}>
-        {backButton && (
-          <IconButton
-            className="navbar__back-button"
-            iconName="arrow_back"
-            id="navbar-back-button"
-            onClick={this.onBackButtonClick}
-          />
-        )}
-        {title && <div className="navbar__title">{title}</div>}
-      </div>
-    );
-  }
-
-  private onBackButtonClick() {
-    if (this.props.onBackButtonClick) {
-      this.props.onBackButtonClick();
-    }
-  }
+function Navbar({
+  backButton,
+  title,
+  className,
+  onBackButtonClick,
+}: NavbarProps) {
+  return (
+    <div className={classNames("navbar", className)}>
+      {backButton && (
+        <IconButton
+          className="navbar__back-button"
+          iconName="arrow_back"
+          id="navbar-back-button"
+          onClick={onBackButtonClick}
+        />
```

Comment:
I've brought it up in a few PRs along the way that I feel like this button should actually be a link to improve accessibility.  Perhaps if `Navbar` accepted a `backLink` prop instead of `onBackButtonClick` and pass it down (I think the underlying `Button` already supports switching out the role.

Reply:
There are too many accessibility problems related to the app buttons. Maybe we should create an issue in the main repo and handle it outside this raid's scope.